features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/site/public/theme/defaultTheme.js

```diff
@@ -1,4 +1,8 @@
 const APP_CONTEXT = '/publisher';
+const REVERSE_PROXY = {
+    enabled: false,
+    origin: 'https://localhost:9443',
+};
```

Comment:
@chanaka3d shouldn't these go to the new settings.js?

Reply:
Yes.. But the file is not there in the publisher app. Ideally we need publisher and store to have the same structure..